charts/federation-v2/values.yaml

```diff
@@ -2,7 +2,7 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-## Configuration values for federation v2 controllermanager statefulset.
+## Configuration values for federation v2 controllermanager deployment.
```

Comment:
Please check https://github.com/kubernetes-sigs/federation-v2/pull/245#discussion_r218828597 for why we are using `sts` here.

/cc @marun

Reply:
thanks for reminding @gyliu513 :) I have checked the comment, and that's in the absence of leader election. We need deploy multi instances with leader election, and to the multi instances of controller, deployment should be fine like kube-controller-manager.